.travis.yml

```diff
@@ -14,8 +20,8 @@ matrix:
 
 env:
   matrix:
-    - BUILD_TYPE="Debug"
-    - BUILD_TYPE="Release"
+    - BUILD_TYPE='Debug'
+    - BUILD_TYPE='Release'
```

Comment:
No variable is searched with single quotes, that is not the case with double quotes.